src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsHostBuilderExtensions.cs

```diff
@@ -135,6 +138,38 @@ public static IHostBuilder ConfigureWebJobs(this IHostBuilder builder, Action<Ho
             return builder;
         }
 
+        /// <summary>
+        /// Configures the specified <see cref="IHostBuilder"/> as a scale manager host.
+        /// </summary>
+        /// <param name="builder">The <see cref="IHostBuilder"/> to configure.</param>
+        /// <param name="configureScaleOptions">Configuration action for <see cref="ScaleOptions"/>.</param>
+        /// <returns>The <see cref="IHostBuilder"/>.</returns>
+        public static IHostBuilder ConfigureWebJobsScale<TConcurrencyStatusRepository, TScaleMetricsRepository>(this IHostBuilder builder, Action<ScaleOptions> configureScaleOptions, string hostId)
```

Comment:
Also need `IAzureBlobStorageProvider` to initialize `BlobStorageConcurrencyStatusRepository`.
Both SC and function runtime need to use `BlobStorageConcurrencyStatusRepository` but `IAzureBlobStorageProvider `should be different.